src/transaction.js

```diff
@@ -69,6 +97,12 @@ Transaction.fromBuffer = function (buffer, __noStrict) {
     })
   }
 
+  if (hasWitnesses) {
+    for (i = 0; i < vinLen; ++i) {
+      tx.inputs[i].witness = readScript()
```

Comment:
Well,  we don't decompile the script here,  so technically  `readScript` is just `readSlice` + a var int.
Shall I change that?